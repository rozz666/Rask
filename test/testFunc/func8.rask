f() -> void
{
    var local = 1;
}

main() -> void
{
    print(local);
}
