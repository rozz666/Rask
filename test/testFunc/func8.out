func8.rask:8:11: error: unknown identifier 'local'

